cpp/unreal_plugins/SpCore/Source/SpCore/Unreal.h

```diff
@@ -428,13 +428,12 @@ class SPCORE_API Unreal
     {
         auto components = Std::toVector<TActorComponent*>(
             getComponentsByType<TActorComponent>(actor) |
-            std::views::transform([&tags](auto component) { return std::make_pair(component, getComponentHasTags(actor, tags)); }) |
+            std::views::transform([&tags](auto component) { return std::make_pair(component, getComponentHasTags(component, tags)); }) |
```

Comment:
Good catch. Also lol how did this code compile? I'm trying to pass an `AActor*` to a function that is expecting a `UActorComponent*`.

Reply:
It did not compile on Linux, but not sure about Windows.